src/ui/screens/settings/ViewPrivacy.qml

```diff
@@ -54,61 +54,27 @@ MZViewBase {
                 }
             }
         }
-        Repeater {
-            id: repeater
-
-            delegate: MZCheckBoxRow {
-                objectName: modelData.objectName
-
-                Layout.fillWidth: true
-                Layout.rightMargin: MZTheme.theme.windowMargin
-                labelText: modelData.settingTitle
-                subLabelText: modelData.settingDescription
-                isChecked: MZSettings.dnsProviderFlags & modelData.settingValue
-                showDivider: false
-                onClicked: {
-                    let dnsProviderFlags = MZSettings.dnsProviderFlags;
-                    dnsProviderFlags &= ~MZSettings.Custom;
-                    dnsProviderFlags &= ~MZSettings.Gateway;
-
-                    if (dnsProviderFlags & modelData.settingValue) {
-                        dnsProviderFlags &= ~modelData.settingValue;
-                    } else {
-                        dnsProviderFlags |= modelData.settingValue;
-                    }
 
-                    // We are not changing anything interesting for the privacy/dns dialog.
-                    if (MZSettings.dnsProviderFlags !== MZSettings.Custom) {
-                        MZSettings.dnsProviderFlags = dnsProviderFlags;
-                        return;
-                    }
+        PrivacyFeaturesList {
```

Comment:
Is this file still used, now that privacy/ViewPrivacy.qml has been introduced?

Reply:
Wondering the same thing

Reply:
Nope! Rebase error, good catch.